x-pack/plugins/security_solution/public/flyout/right/components/analyzer_preview.tsx

```diff
@@ -58,24 +59,36 @@ export const AnalyzerPreview: React.FC = () => {
     [cache.statsNodes]
   );
 
-  if (!documentId || !index || !items || items.length === 0) {
-    return null;
-  }
+  const showAnalyzerTree = documentId && index && items && items.length > 0 && !error;
 
-  return (
-    <div data-test-subj={ANALYZER_PREVIEW_TEST_ID}>
-      <EuiTreeView
-        items={items}
-        display="compressed"
-        aria-label={i18n.translate(
-          'xpack.securitySolution.flyout.right.visualizations.analyzerPreview.treeViewAriaLabel',
-          {
-            defaultMessage: 'Analyzer preview',
-          }
-        )}
-        showExpansionArrows
-      />
-    </div>
+  return loading ? (
+    <EuiSkeletonText
+      data-test-subj={ANALYZER_PREVIEW_LOADING_TEST_ID}
+      contentAriaLabel={i18n.translate(
+        'xpack.securitySolution.flyout.right.visualizations.analyzerPreview.loadingAriaLabel',
+        {
+          defaultMessage: 'analyzer preview',
```

Comment:
what do you think about changing this defaultMessage to `Loading analyzer preview`?